app/results/search_dropdown.jsx

```diff
@@ -0,0 +1,74 @@
+'use client';
+
+import { useEffect, useState } from 'react';
+import product_data from './search_bar_data/MOCK_DATA.json';
+import { useSearchParams, usePathname, useRouter } from 'next/navigation';
+
+const Search_Dropdown = ({ query, isFocused, setIsFocused }) => {
+  const [suggestions, setSuggestions] = useState([]);
+  const searchParams = useSearchParams();
+  const router = useRouter();
+
+  // Uses debounce to ensure not updating on every letter input
```

Comment:
Very nice idea to use debounce! ❤️

Reply:
I ended up not using debounce in the end, I just forgot to remove that comment, but Bryant wanted the functionality on submit instead of on change to route to another page, so there was no need to delay the search call on an api.

Reply:
Speaking with Aric and this didn't need debounce. Behavior changed mid development, so it's a comment that can be deleted.